flair/embeddings.py

```diff
@@ -734,6 +734,10 @@ def __init__(self,
         """
         super().__init__()
 
+        if bert_model not in ['bert-base-uncased', 'bert-large-uncased', 'bert-base-cased', 'bert-large-cased',
+                              'bert-base-multilingual-cased', 'bert-base-multilingual-uncased', 'bert-base-chinese']:
+            raise ValueError('Provided bert-model is not available.')
```

Comment:
I think it would be better to use the internal `PRETRAINED_MODEL_ARCHIVE_MAP` variable from the `pytorch_pretrained_bert` module, see [here](https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/modeling.py#L41) :)